streamer/src/nonblocking/quic.rs

```diff
@@ -99,8 +99,10 @@ pub fn spawn_server(
     coalesce: Duration,
 ) -> Result<(Endpoint, Arc<StreamStats>, JoinHandle<()>), QuicServerError> {
     info!("Start {name} quic server on {sock:?}");
-    let (config, _cert) = configure_server(keypair, gossip_host)?;
-
+    let (mut config, _cert) = configure_server(keypair, gossip_host)?;
+    config.concurrent_connections(
+        max_staked_connections.saturating_add(max_unstaked_connections) as u32,
+    );
```

Comment:
shouldn't this go in `configure_server` along with other configs there?

Reply:
good idea

Reply:
Done